lib/rhykane/transformer.rb

```diff
@@ -25,6 +25,10 @@ def nest(row, *args)
 
         Dry::Transformer::HashTransformations.nest(row, key, *vals)
       end
+
+      def set_default(row, default)
+        row.to_h.tap { |r| default.each { |k, v| r[k] ||= v } }
```

Comment:
`default.merge(row)`?

Reply:
I've tried a few different variations of this but the nil values don't get replaced with their defaults!

Reply:
ah right